src/main/java/picard/analysis/artifacts/CollectSequencingArtifactMetrics.java

```diff
@@ -332,13 +324,42 @@ protected void finish() {
                     baitBiasDetailMetricsFile.addMetric(baitBiasDetailMetrics);
                 }
             }
-
         }
 
         preAdapterDetailMetricsFile.write(preAdapterDetailsOut);
         preAdapterSummaryMetricsFile.write(preAdapterSummaryOut);
         baitBiasDetailMetricsFile.write(baitBiasDetailsOut);
         baitBiasSummaryMetricsFile.write(baitBiasSummaryOut);
+
+        // Calculate the summary error rates - it's CRITICAL that the other files are written out
+        // first as this code modifies the pre-adapter detail metrics!
+        if (!preAdapterDetailMetricsFile.getMetrics().isEmpty()) {
+            final List<PreAdapterDetailMetrics> in = preAdapterDetailMetricsFile.getMetrics();
+            in.forEach(m -> {
+                if (m.REF_BASE == 'G' || m.REF_BASE == 'T') {
+                    m.REF_BASE = (char) SequenceUtil.complement((byte) m.REF_BASE);
+                    m.ALT_BASE = (char) SequenceUtil.complement((byte) m.ALT_BASE);
```

Comment:
shouldn't the counts be negated if you complement the bases? I'm confused about what this is trying to do otherwise...